vars/uploadArtifacts.groovy

```diff
@@ -0,0 +1,26 @@
+void call(Map args = [:]) {
+    lib = library(identifier: "jenkins@current", retriever: legacySCM(scm))
+
+    def manifestFilename = args.manifest ?: 'builds/manifest.yml'
+    def buildManifest = lib.jenkins.BuildManifest.new(readYaml(file: manifestFilename))
+
+    def artifactPath = buildManifest.getArtifactRoot("${JOB_NAME}", "${BUILD_NUMBER}")
+    echo "Uploading to s3://${ARTIFACT_BUCKET_NAME}/${artifactPath}"
+
+    withAWS(role: 'opensearch-bundle', roleAccount: "${AWS_ACCOUNT_PUBLIC}", duration: 900, roleSessionName: 'jenkins-session') {
+        if (args.upload) {
```

Comment:
upload->dryrun?

Reply:
I kinda wanted it to be `false` by default as a general rule of thumb, but it's not a pattern (yet). WDYT?